server/views/partials/prisonerSearchTable.njk

```diff
@@ -0,0 +1,46 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% if (prisoners and prisoners.length > 0) %}
+<table id="reports-table" class="govuk-table" data-qa="reports-table">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header"></th>
+        <th scope="col" class="govuk-table__header">Name</th>
+        <th scope="col" class="govuk-table__header">Prisoner ID</th>
+        <th scope="col" class="govuk-table__header">Prison</th>
+        <th scope="col" class="govuk-table__header">Cell location</th>
+        <th scope="col" class="govuk-table__header"></th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body govuk-body-s">
+    {% for prisoner in prisoners %}
+      {% set rowClass = 'even' if loop.index % 2 === 0 else 'odd' %}
```

Comment:
This can probably be done directly in SASS using `tr:nth-child()`. I know this is just a spike though so it doesnt matter